canvas_modules/common-canvas/src/common-properties/components/control-item/control-item.scss

```diff
@@ -15,7 +15,12 @@
  */
 
 .properties-control-item {
-	padding: $spacing-04 0;
+	&.parent-component { // When control-item displays other controls
```

Comment:
Why do we need to have different padding here?

Reply:
When checkboxset is called from control-item **(existing code)** -
Padding is added above and below checkboxset control.
![Screen Shot 2020-12-03 at 11 09 06 AM](https://user-images.githubusercontent.com/25124000/101077083-27bd2200-3559-11eb-9229-0e78472a2106.png)

When control-item is called from checkboxset **(new code)** -
Padding is added above and below label. This adds extra space between label and checkboxset. To avoid this, I removed bottom padding for this case.
![Screen Shot 2020-12-03 at 11 13 51 AM](https://user-images.githubusercontent.com/25124000/101077099-2ee43000-3559-11eb-93c4-e9a848545664.png)